dateutil/parser/_parser.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, instream):
         self.charstack = []
         self.tokenstack = []
         self.eof = False
+        self.join_numeric_separated_by_comma = join_numeric_separated_by_comma
```

Comment:
can you add a comment for why this option exists, namely for the python logging format

Reply:
Is it just the python logging format? I think that it's *also* for the `ISO-8601` format, which *technically* (though not in practice) prefers comma as the fractional separator.

Reply:
Oh neat, I had no idea.  I guess I've seen that used in conjunction with Euros, e.g. "€ 29.123,45"

Does the ISO usage need to be for any numbers or is it just for fractional seconds?  i.e. will the check-for-colons above work?

Reply:
Technically it's for all fractional time components, but I think basically no one uses anything except fractional seconds. Still, this is technically a valid ISO-8601 datetime:

`2014-01-01T00,50` -> `datetime(2014, 1, 1, 0, 30)`